x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_weblogs/ml/url_scanning.json

```diff
@@ -1,13 +1,14 @@
 {
-  "groups": ["kibana_sample_logs", "kibana_sample_data"],
-  "description": "Kibana sample web logs data: find anomalies in the distinct count of client IPs",
+  "groups": ["kibana_sample_logs", "kibana_sample_web_logs"],
```

Comment:
As above, we were planning on putting all the jobs for all the sample modules into the group `kibana_sample_data` as well as their own weblogs specific group. So I think this should be 
```
"groups": ["kibana_sample_web_logs", "kibana_sample_data"],
```  
(if you prefer `kibana_sample_web_logs` to `kibana_sample_logs` that was there before).